qutebrowser/browser/pdfjs.py

```diff
@@ -49,7 +49,8 @@ def generate_pdfjs_page(filename, url):
         filename: The filename of the PDF to open.
         url: The URL being opened.
     """
-    if not is_available():
+    pdfjs_bin = is_available()
+    if not pdfjs_bin:
```

Comment:
```suggestion
    if pdfjs_bin is None:
```

while we should never get an empty string here, I still think it's better to be explicit about `None` checks.

Reply:
That oversight has been fixed and it should be more pythonic